src/components/Timestamp/Timestamp.tsx

```diff
@@ -54,6 +64,15 @@ export const Timestamp = ({
 			<time dateTime={isoDateTime.toString()} css={timeStyles}>
 				{timeAgo}
 			</time>
+
+			<span css={anchorStyles} className="comment-permalink-icon">
```

Comment:
Not quite sure if theres a way to apply the hover behaviour without using a className unless I extract the css styling out of this class.

Reply:
I think the way to do this would be 

```css
&::hover {
    display: inline-block;
}```

Reply:
Generally we try to avoid using any class names so hopefully we can figure out a way for this!

Reply:
[Potential fix](https://github.com/emotion-js/emotion/issues/1771)?